pkgs/tools/filesystems/ceph/default.nix

```diff
@@ -83,27 +103,30 @@ let
     ps.virtualenv
     # Libraries needed by the python tools
     ps.Mako
+    ceph-common
     ps.cherrypy
+    ps.dateutil
+    ps.jsonpatch
     ps.pecan
     ps.prettytable
     ps.pyjwt
     ps.webob
     ps.bcrypt
+    ps.scipy_1_3
```

Comment:
From the PR description:

> ceph-mgr hangs during startup due to incompatibility with newer scipy versions, thus the version downgrade. Thanks @srhb.

Please put a comment on this here. What is known about this bug? Is there an upstream issue?

I tried to upgrade our Ceph (packaged via a separate derivation), noticed the `ceph-mgr` hang, but did not figure out how to fix it. How did you find out @srhb?

Reply:
Upstream issues are here: https://tracker.ceph.com/issues/42764 and here: https://tracker.ceph.com/issues/45147. AFAIK, the current official upstream solution is to disable the diskprediction_local module if python is >= 3.8 instead of trying to fix the actual problem (https://github.com/ceph/ceph/pull/34846/files), since Python 3.8 is not officially supported. However, @srhb figured that downgrading scipy made diskprediction_local runnable. Better wait for her to give an explanation on that part :)

Reply:
@nh2 

I can add a comment with the links on monday.

I figured it out with strace on the mgr threads and noticing that one of them was stuck deep in scipy land. From there, I found the above issues, which upstream doesn't seem terribly keen on fixing, because RHEL/Ubuntu don't use a new enough python/scipy to even experience the problem yet.

As such, I consider the scipy downgrade our best path, because local disk prediction is a potentially nice feature to have, and hopefully upstream will become compatible in time so that we can drop the old scipy. I'll comment to that effect in the pr. :smile:

Reply:
er, not strace, gdb*

Reply:
We could also just flip to python 3.7 here instead of 3.8, couldn't we? It's probably less maintenance overhead than manually verifying compatibility with each python module.

Reply:
I'm not sure I actually tested that, but I can certainly try and see if it works. :+1:

Reply:
No go with `pythonPackages37` -- and with `pythonPackages36` I run into dependency hell with a load of other deps. I'm inclined to keep this solution. I've added the issue links inline, PTAL. :smile:

Reply:
Fair enough.

Reply:
Thanks! :+1: